setup.py

```diff
@@ -314,19 +316,27 @@ def parse_jsargs():
         print("Adding '--build_js' required for 'sdist'")
         sys.argv.append('--build_js')
 
-# check for package install, add "--install_js" to skip prompt
+# check for package install, set jsinstall to False to skip prompt
+jsinstall = True
 if not exists(join(ROOT, 'MANIFEST.in')):
-    installing = any(arg in sys.argv for arg in ('install', 'develop', 'sdist', 'egg_info'))
-    if installing and "--install_js" not in sys.argv and "--build_js" not in sys.argv:
-        print("Adding '--install_js' default for sdist package install")
-        sys.argv.append('--install_js')
-
-jsbuild = parse_jsargs()
+    options = ('install', 'develop', 'sdist', 'egg_info', 'build', 'clean', '--help')
+    installing = any(arg in sys.argv for arg in options)
+    if installing:
+        print("Avoid building or installing JS when BokehJS is not available")
```

Comment:
This message seems really confusing, I'm not entirely sure I understand what it is trying to say. It reads as an imperative to the reader, but I think it's trying to say these options are not available because the BokehJS source is not available? I do also think we need to be careful to distinguish. My personal feeling is that "BokehJS" refers to the build "bokeh.min.js" (and I think this matches my observation of how other people use the term) and s if we mean the source code we should be very explicit about that and say "BokehJS source code". I'd suggest:
```
BokehJS source code is not shipped in sdist packages; building/installing from the bokehjs 
source directory is disabled. To build or develop BokehJS yourself, you must clone the full 
Bokeh repository from https://github.com/bokeh/bokeh"
```